src/coreclr/jit/emitarm64.cpp

```diff
@@ -24488,6 +24616,10 @@ void emitter::getMemoryOperation(instrDesc* id, unsigned* pMemAccessKind, bool*
                 }
                 break;
             case IF_LARGELDC:
+            case IF_SVE_ID_2A:
```

Comment:
we are not tracking this currently for any of the SVE formats. we should remove it and add it later when we start tracking it.

Reply:
If I remove it, I'll also need to remove the LD and ST flags from the table for those instructions otherwise we will hit an assert.

Reply:
I would suggest to remove them, because pretty sure we need to handle it at other places and not just in this method. So lets do it together.

Reply:
Sounds good to me!